sdk/communication/Azure.Communication.CallAutomation/src/Models/Events/ReasonCodeName.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System;
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary>
+    /// Failure Reason for incoming webhook events.
+    /// </summary>
+    public readonly partial struct ReasonCodeName : IEquatable<ReasonCodeName>
+    {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="ReasonCodeName"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public ReasonCodeName(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string RecognizeInitialSilenceTimeoutValue = "8510";
+        private const string RecognizeInterDigitTimeoutValue = "8532";
+        private const string RecognizePlayPromptFailedValue = "8511";
+
+        private const string RecognizeMaxDigitsReceivedValue = "8531";
+        private const string RecognizeStopToneDetectedValue = "8514";
+
+        private const string PlayDownloadFailedValue = "8536";
+        private const string PlayOperationCancelledValue = "8508";
```

Comment:
Do we need this `PlayOperationCancelledValue` ? Since we  already have s separate event `OperationCanceledValue` now?

Reply:
Good catch, thank you. It was a left over of the refactoring.